pkg/csi/service/vanilla/controller.go

```diff
@@ -1790,6 +1788,7 @@ func (c *controller) processQueryResultsListVolumes(ctx context.Context, startin
 						log.Errorf("Failed to get node vm object from the node name, err:%v", err)
 						return entries, nextToken, volumeType, err
 					}
+					volCounter += 1
```

Comment:
will volCounter be incremented multiple times for same volume id published on multiple nodes (RWM access mode) here? shall we increment it only once per volume id?

Reply:
This method getBlockVolumeToHostMap is used only for block volumes(RWO). Not for file volumes.